neurons/validator.py

```diff
@@ -480,10 +482,25 @@ async def check_videos_and_calculate_rewards(
                 embeddings.video, query_emb
             ).tolist()
 
+            # Compute if there are penalties for random descriptions
+            are_descriptions_valid = [imagebind_desc_mlp.is_desc_embedding_valid(embedding) for embedding in embeddings.description]
+            description_penalties = []
+            # Apply penalties and store the penalized scores
+            for desc_score, desc_valid in zip(description_relevance_scores, are_descriptions_valid):
+                if not desc_valid:
+                    penalized_score = (
+                        (desc_score) -
+                        (desc_score * (1 - RANDOM_DESCRIPTION_PENALTY))
+                    ) * -1
```

Comment:
this equation can be simplified to `penalized_score = (desc_score * RANDOM_DESCRIPTION_PENALTY) * -1`. i think that's nicer for readability's sake